### ISSUE TITLE
NX Workspace solution-style TS config error on schematic

### ISSUE THREAD

**Issue by KingDarBoja** [angular-eslint/angular-eslint#191]
Hello @JamesHenry 

Just tried to run the schematic on some nx workspace which still uses Angular 10 and ran into an error due to solution-style TypeScript config.

```console
$ ng add @angular-eslint/schematics

>  NX   WARNING  The Angular CLI was invoked instead of the Nx CLI. Use "npx ng [command]" or "nx [command]" instead.

Your global Angular CLI version (11.0.2) is greater than your local version (10.2.0). The local Angular CLI version is used.

To disable this warning use "ng config -g cli.warnings.versionMismatch false".
Installing packages for tooling via yarn.
Installed packages for tooling via yarn.

Error: Angular CLI v10.1.0 and later (and no `tsconfig.base.json`) is required in order to run this schematic. Please update your workspace and try again
```

Not sure if worthy take a look but NX Workspaces have an option to pick current Angular TSLint on creation and probably several projects are willing to migrate into ESLint 🚀

**Comment by LayZeeDK** [comment]
Here's the events that caused this:
1. Angular CLI 10.0 introduced solution-style TypeScript configurations.
2. Angular CLI 10.1 rolled back solution-style TypeScript configurations.
3. Nx CLI created a slightly different structure of solution-style TypeScript configurations throughout an Nx workspace instead of rolling them back.

**Comment by JamesHenry** [comment]
I have specifically added notes on the relationship between this tooling and Nx: https://github.com/angular-eslint/angular-eslint#usage-with-nx-monorepos

Proper Nx support for Angular + ESLint is imminent (I worked on it myself), but didn't quite make it into 10.4

Thanks!
